aksel.nav.no/website/pages/templates/soknad-introside/intro-med-alternativ-soknad.tsx

```diff
@@ -0,0 +1,238 @@
+                denne søknaden. Bruk punktliste for god lesbarhet.
+              </BodyLong>
+              <Heading level="3" size="medium" spacing>
+                Dette kan du ikke søke om her
+              </Heading>
+              <BodyLong spacing>
+                Deretter skal det stå informasjon om hva du ikke kan søke om i
+                denne søknaden, og link til alternativ søknad. Bruk punktliste
+                for god lesbarhet.
+              </BodyLong>
+            </Box>
+            <Box>
+              <Heading level="2" size="large" spacing>
+                Før du søker
+              </Heading>
+              <BodyLong spacing>
+                Her skal det kun stå informasjon brukeren vil ha stor nytte av
+                før søknaden. Informasjon brukeren ikke trenger før start, og
+                som brukeren vil få underveis, skal ikke stå her. Undersøk
+                hvilken informasjon som ligger i søknadsdialogen og på
+                kvitteringssiden, slik at vi minimerer dobbelt innhold.
+                Eksempler på nyttig informasjon nedenfor - punktene tas bare med
+                om de er relevante:
+              </BodyLong>
+              <List>
+                <List.Item>
+                  Ting brukeren må ha gjort før søknaden.{" "}
+                  <i>
+                    Du må ha meldt deg som arbeidssøker før du kan søke om
+                    dagpenger.
+                  </i>
+                </List.Item>
+                <List.Item>
+                  Dokumentasjon brukeren kan bli bedt om.{" "}
+                  <i>
+                    Noen av opplysningene du gir underveis vil du bli bedt om å
+                    dokumentere. Du vil trenge xx og xx for å fullføre denne
+                    søknaden.
+                  </i>
+                </List.Item>
+                <List.Item>
+                  Automatisk lagring.{" "}
+                  <i>
+                    Vi lagrer svarene dine (xx timer) mens du fyller ut, så du
+                    kan ta pauser underveis.
+                  </i>
+                </List.Item>
+                <List.Item>
+                  Antall steg og estimert tidsbruk.{" "}
+                  <i>
+                    Det er XX steg i søknaden, og du kan regne med å bruke ca.
+                    XX minutter.
+                  </i>
+                </List.Item>
+                <List.Item>
+                  Søknadsfrister.{" "}
+                  <i>Husk at du må søke om xx innen xx dager.</i>
+                </List.Item>
+                <List.Item>
+                  Saksbehandlingstider og info om gyldighet, krav osv.{" "}
+                  <i>
+                    Vi bruker ca. xx uker på å behandle søknaden din. Husk at du
+                    må sende meldekort xx ofte selv om du ikke har fått svar på
+                    søknaden din om dagpenger ennå.
+                  </i>
+                </List.Item>
+              </List>
```

Comment:
Etter listen (fortsatt under samme overskrift) kan du legge til denne teksten:
For annen, utfyllende informasjon om søknaden bør du lenke direkte til søknadskapittelet i produktsiden, som [dette eksempelet for dagpenger](https://www.nav.no/dagpenger#sok).